server/command.go

```diff
@@ -292,56 +291,44 @@ func (p *Plugin) SendLinksWithDetails(userID, channelID string, links []*storemo
 	sb.WriteString("| Mattermost Team | Mattermost Channel | MS Teams Team | MS Teams Channel | \n| :------|:--------|:-------|:-----------|")
 	errorsFound := false
 	wg := sync.WaitGroup{}
-	batchSize := math.Max(math.Sqrt(float64(len(links))), 1000)
 
-	var msTeamsTeamIDsVsNames sync.Map
+	msTeamsTeamIDsVsNames := make(map[string]string)
 	var msTeamsChannelIDsVsNames sync.Map
```

Comment:
Also we no longer need here the `sync.Map`, it is going to be simpler and more efficient to use a regular map.